docs/sce_file_contents.md

```diff
@@ -133,17 +133,32 @@ CITE-seq data, when present, is included within the `SingleCellExperiment` as an
 altExp(sce, "CITEseq")
 ```
 
-Within this, the main expression matrix is again found in the `counts` assay, with each column corresponding to a cell or droplet (in the same order as the parent `SingleCellExperiment`) and each row corresponding to an antibody derived tag (ADT).
-Column names are again cell barcode sequences and row names the antibody targets for each ADT.
+Within this, the main expression matrix is again found in the `counts` assay and the normalized expression matrix is again found in the `logcounts` assay.
+For each assay, each column corresponds to a cell or droplet (in the same order as the parent `SingleCellExperiment`) and each row corresponds to an antibody derived tag (ADT).
+Column names are again cell barcode sequences and row names are the antibody targets for each ADT.
+
 
 The following additional per-cell data columns for the CITE-seq data can be found in the main `colData` data frame (accessed with `colData(sce)` [as above](#cell-metrics)).
 
 | Column name                | Contents                                          |
 | -------------------------- | ------------------------------------------------- |
-| `altexps_CITEseq_sum  `    | UMI count for CITE-seq ADTs                       |
+| `altexps_CITEseq_sum`    | UMI count for CITE-seq ADTs                       |
 | `altexps_CITEseq_detected` | Number of ADTs detected per cell (ADT count > 0 ) |
 | `altexps_CITEseq_percent`  | Percent of `total` UMI count from ADT reads       |
 
+
+The following columns, representing QC statistics from [`DropletUtils::cleanTagCounts()`](https://rdrr.io/github/MarioniLab/DropletUtils/man/cleanTagCounts.html), can be further be found in the `"CITEseq"`  alternative experiment `colData`, accessed with `colData(altExp(sce, "CITEseq"))`.
+
+| Column name                | Contents                                          |
+| -------------------------- | ------------------------------------------------- |
+| `zero.ambient`   | Indicates whether the cell has zero ambient contamination   |
+| `sum.controls` |  The sum of counts for all control features; Only present if negative control ADTs are present. |
+| `high.controls`  | Indicates whether the cell has unusually high total control counts; Only present if negative control ADTs are present.|
+| `ambient.scale` |  The relative amount of ambient contamination; Only present if negative control ADTs are _not_ present. |
+| `high.ambient`  | Indicates whether the cell has unusually high contamination; Only present if negative control ADTs are _not_ present.|
+| `discard`  | Indicates whether the cell should be discarded based on QC statistics; All retained cells in the `_processed.rds` are `FALSE`. |
```

Comment:
I don't know if we need the second sentence here; I feel like it may be covered by the discussion at `scpca_filter`, but it doesn't hurt too much.

Reply:
I didn't delete this, but still can!